### PR TITLE
Update c.vim

### PR DESCRIPTION
**Pull request by bleemayer**
Added C common socket names.

### CODE REVIEW

File: after/syntax/c.vim [hunk 1]
```diff
@@ -232,6 +232,11 @@ syn keyword	cAnsiName	STDC CX_LIMITED_RANGE
 syn keyword	cAnsiName	STDC FENV_ACCESS
 syn keyword	cAnsiName	STDC FP_CONTRACT
 
+syn keyword cAnsiName AF_INET SOCK_STREAM INADDR_ANY AF_INET
+235 syn keyword cAnsiName SOL_SOCKET SO_REUSEPORT SO_REUSEADDR
+236 syn keyword cAnsiName SO_RCVTIMEO IPPROTO_TCP TCP_NODELAY
+237 syn keyword cAnsiName SOCK_DGRAM POLLIN
```
Comment (justinmk) [inline after/syntax/c.vim]: What's with the numbers?

**Comment by bleemayer** [comment]
My mistake, sorry, I forgot to delete, they are line number that came when I copied them. I corrected and commited. Now the pull request is correct.

Regards,

Brian

File: after/syntax/c.vim [hunk 2]
```diff
@@ -233,9 +233,9 @@ syn keyword	cAnsiName	STDC FENV_ACCESS
 syn keyword	cAnsiName	STDC FP_CONTRACT
 
 syn keyword cAnsiName AF_INET SOCK_STREAM INADDR_ANY AF_INET
-235 syn keyword cAnsiName SOL_SOCKET SO_REUSEPORT SO_REUSEADDR
-236 syn keyword cAnsiName SO_RCVTIMEO IPPROTO_TCP TCP_NODELAY
-237 syn keyword cAnsiName SOCK_DGRAM POLLIN
+syn keyword cAnsiName SOL_SOCKET SO_REUSEPORT SO_REUSEADDR
+syn keyword cAnsiName SO_RCVTIMEO IPPROTO_TCP TCP_NODELAY
+syn keyword cAnsiName SOCK_DGRAM POLLIN
```
Comment (justinmk) [inline after/syntax/c.vim]: Are these really ANSI C, or POSIX?

Not sure it's a good idea to start adding "common" things instead of just "ansi" things.

Comment (bleemayer) [reply after/syntax/c.vim]: They are POSIX :-(

Comment (bleemayer) [reply after/syntax/c.vim]: Do you have group cPosixName? I could add to that instead.

**Pull request merged**